utils/log.sh

```diff
@@ -178,20 +178,32 @@ overwrite() {
 # - $2 : error message (eg. "Failed to remove elastic-search service")
 try() {
     local -r COMMAND=${1:?"FATAL: function 'try' is missing a parameter"}
-    local -r ERROR_MESSAGE=${2:?"FATAL: function 'try' is missing a parameter"}
+    local -r ERROR_MESSAGE=${2:-""}
```

Comment:
I think we should always log an error, even if we don't want to exit. We should add another parameter to specify if you **don't** want to exit, as in, have a third parameter that defaults to true. With $3 == true, log error and exit 1, with $3 -- false, log error.

Reply:
The message then will be obligatory which is not the case ?

Reply:
I think it being obligatory is best.